src/cards/communityChestDeck.js

```diff
@@ -0,0 +1,102 @@
+inherits = require('util').inherits;
+Deck = require("./deck");
+var Card = require("./card.js");
+
+
+function CommunityChestDeck() {
+    Deck.call(this);
+    this.cards = buildCommunityChestDeck();
+};
+
+inherits(CommunityChestDeck, Deck);
+
+var buildCommunityChestDeck = function() {
+
+    var card1 = new Card("Advance to Go (Collect $200)", function(game) {
+        game.getCurrentPlayer().moveTo(0);
+    });
+
+    var card2 = new Card("Bank error in your favor - collect $75", function(game) {
+        game.getCurrentPlayer().makeDeposit(75);
+    });
+
+    var card3 = new Card("Doctor's fees - Pay $50", function(game) {
+        game.getCurrentPlayer().makePayment(50);
+    });
+
+    var card4 = new Card("Get out of jail free - this card may be kept until needed, or sold", function(game) {
+        game.getCurrentPlayer().getOutOfJailFreeCards += 1;
+    });
+
+    var card5 = new Card("Go to jail - go directly to jail - Do not pass Go, do not collect $200", function(game) {
+        game.getCurrentPlayer().sendToJail();
+    });
+
+    var card6 = new Card("It is your birthday Collect $10 from each player", function(game) {
+        game.getCurrentPlayer().collectFromPlayers(10, game.players);
+    });
+
+    var card7 = new Card("Grand Opera Night - collect $50 from every player for opening night seats", function(game) {
+        game.getCurrentPlayer().collectFromPlayers(50, game.players);
+    });
+
+    var card8 = new Card("Income Tax refund - collect $20", function(game) {
+        game.getCurrentPlayer().makeDeposit(20);
+    });
+
+    var card9 = new Card("Life Insurance Matures - collect $100", function(game) {
+        game.getCurrentPlayer().makeDeposit(100);
+    });
+
+    var card10 = new Card("Pay Hospital Fees of $100", function(game) {
+        game.getCurrentPlayer().makePayment(100);
+    });
+
+    var card11 = new Card("Pay School Fees of $50", function(game) {
+        game.getCurrentPlayer().makePayment(50);
+    });
+
+    var card12 = new Card("Receive $25 Consultancy Fee", function(game) {
+        game.getCurrentPlayer().makeDeposit(25);
+    });
+
+    var card13 = new Card("You are assessed for street repairs - $40 per house, $115 per hotel", function(game) {
```

Comment:
repeated code here but i think it might confuse things more to abstract this to card or something so maybe it's ok